### DISCUSSION TITLE
What's the behavior of a source repository without --default and --secondary ?

### DISCUSSION THREAD

**Discussion by amarion35** [python-poetry/poetry#7195] [Q&A] [answered]
If I run `poetry source add --help` I get these lines:

      Options:
        -d, --default         Set this source as the default (disable PyPI). A default source will also be the fallback source if you add other sources.
        -s, --secondary       Set this source as secondary.

So we know what happen with each one of these options but we don't know exactly what will be the real "default" behavior without any options or with both.

PS: I'm looking forward to #6879 which will clarify the types of sources.

**Comment by neersighted** [comment, marked as answer]
You can refer to the source docs, or merely the comment in the linked issue, https://github.com/python-poetry/poetry/issues/6713#issue-1398042565. A 'primary' (no options) source is consulted before PyPI.
